orcoursetrion/tests/base.py

```diff
@@ -151,17 +213,57 @@ def register_team_list(self, body):
             body=body
         )
 
+    def register_team_create(self, body):
+        """
+        Create team URL/method
+        """
+        httpretty.register_uri(
+            httpretty.POST,
+            '{url}orgs/{org}/teams'.format(
+                url=self.URL,
+                org=self.ORG,
+            ),
+            body=body
+        )
+
+    def register_team_members(self, body):
+        """
+        Team membership list API.
+        """
+        httpretty.register_uri(
+            httpretty.GET,
+            re.compile(
+                '^{url}teams/\d+/members$'.format(
+                    url=re.escape(self.URL)
+                )
+            ),
+            body=body
+        )
+
+    def register_team_memberhip(self, body):
```

Comment:
zomg, thank you.  I tried to figure this out for like 15 minutes and gave up by just copy pasting the function name as I couldn't figure out why it kept erroring out on the name `register_team_membership` doesn't exist.  I was clearing pyc files and everything lol.